README.adoc

```diff
@@ -162,7 +163,9 @@ When a role is mapped to the `ALL_AUTHENTICATED_USERS` special subject, then any
 === Encoding passwords using securityUtility
 
 It is not recommended to store passwords in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encode the passwords, as you can see in the `userRegistry.xml`. 
-For example, to encode 'bobpwd', the Liberty `securityUtility` command can be run from the `target/liberty/wlp/bin` directory after you have installed Liberty.
+For example, to encode 'bobpwd', the Liberty `securityUtility` command can be run from the `wlp/bin` from the Liberty application server.
```

Comment:
There're two "from" in this sentence. Add directory to the`wlp/bin` directory.

Reply:
Ok, I added it in the last commit.